nvflare/edge/tools/edge_recipe.py

```diff
@@ -91,6 +151,49 @@ def __init__(
 
 
 class EdgeRecipe(Recipe):
+    """Recipe class for cross-edge federated learning using NVFlare's hierarchical edge system.
+
+    This class provides a high-level interface for configuring cross-edge
+    federated learning jobs. It configures the necessary components
+    including model managers, device managers, evaluators, and device simulation settings.
+
+    The recipe supports both real device connections and simulated device training,
+    making it suitable for both production deployment and prototyping/testing.
+
+    Example usage:
+        ```python
+        # Basic configuration
+        model_manager_config = ModelManagerConfig(
+            global_lr=0.1,
+            num_updates_for_model=20,
+            max_model_version=300,
+            max_model_history=100
+        )
+
+        device_manager_config = DeviceManagerConfig(
+            device_selection_size=200,
+            min_hole_to_fill=10,
+            device_reuse=False
+        )
+
+        recipe = EdgeRecipe(
+            job_name="my_edge_job",
+            model=MyModel(),
+            model_manager_config=model_manager_config,
+            device_manager_config=device_manager_config
+        )
+        ```
+
+    Attributes:
+        job_name: Name of the federated learning job
+        method_name: Method type, always set to "edge"
```

Comment:
maybe we should instead explain the init method's argument.
this "method_name" let's not expose it, this might be removed.

Reply:
removed